core/src/main/scala/sbt/librarymanagement/LibraryManagement.scala

```diff
@@ -0,0 +1,198 @@
+package sbt.librarymanagement
+
+import java.io.File
+import sbt.util.Logger
+import sbt.io.Hash
+import sbt.librarymanagement.syntax._
+
+/**
+ * Helper mixin to provide methods for library management
+ */
+abstract class LibraryManagement extends LibraryManagementInterface {
```

Comment:
(Forwarding my review of #123) Please could we define two different entities and use composition over inheritance?